src/components/icon-dropdown/index.js

```diff
@@ -74,7 +75,7 @@ class Dropdown extends Component {
                     />
                 {visible &&
                     <div data-focus='dropdown-menu' data-position={openDirection} ref='dropdown'>
-                        {operationList.map(({label, action}, idx) => (<div key={idx} data-role='dropdown-item' onClick={this._operationActionWrapper(action)}>{label}</div>))}
+                        {operationList.map(({label, action}, idx) => (<div key={idx} data-role='dropdown-item' onClick={this._operationActionWrapper(action)}>{i18n.t(label)}</div>))}
```

Comment:
I didn't see at first, but here it's a real class, not a component. So instead of using `i18nMixin`, you might want to use the `TranslationBehaviour`, available [here](https://github.com/KleeGroup/focus-components/blob/develop/src%2Fbehaviours%2Ftranslation%2Findex.js). You can see an example [here](https://github.com/KleeGroup/focus-components/blob/develop/src%2Fcomponents%2Fpanel%2Findex.js#L19)